include/uv-win.h

```diff
@@ -366,8 +366,8 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
     struct {                                                                  \
       OVERLAPPED overlapped;                                                  \
       size_t queued_bytes;                                                    \
-    };                                                                        \
-  };                                                                          \
+    } io;                                                                     \
+  } req;                                                                      \
```

Comment:
Maybe call this one 'data'? Otherwise we end up with some "req.req.xyz" code, which looks like a mistake, but it's not.